taskiq/cli/async_task_runner.py

```diff
@@ -277,3 +280,4 @@ async def async_listen_messages(  # noqa: C901, WPS210, WPS213
             await broker.result_backend.set_result(message.task_id, result)
         except Exception as exc:
             logger.exception(exc)
+            raise ResultSetError() from exc
```

Comment:
I think it would be better not to raise exception here. Since it will kill the worker's process.

Reply:
Okay, but maybe we can change logger message to more understandable? I think that people need to explicitly understand that the broker is broken and cannot accept the message